phpBB/phpbb/captcha/plugins/qa.php

```diff
@@ -674,11 +680,10 @@ function acp_page($id, &$module)
 		else
 		{
 			// okay, show the editor
-			$error = false;
-			$input_question = request_var('question_text', '', true);
-			$input_answers = request_var('answers', '', true);
-			$input_lang = request_var('lang_iso', '', true);
-			$input_strict = request_var('strict', false);
+			$input_question = $this->request->variable('question_text', '', true);
+			$input_answers = $this->request->variable('answers', '', true);
+			$input_lang = $this->request->variable('lang_iso', '', true);
```

Comment:
This is duplicate with `acp_get_question_input()`
Please use the method instead (this also cleans the answers list, in case you forgot to add an answer)